react/src/pages/NotificationPopover.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles, Paper, Box, Typography, Tooltip, IconButton, Popover } from '@material-ui/core';
+import { NotificationsActive } from '@material-ui/icons';
+import Notification from './Notification';
+import { AnalysisRow } from './analysis/Analysis';
+
+const useStyles = makeStyles(theme => ({
+    popover: {
+        minWidth: '450px',
+    },
+    paper: {
+        padding: theme.spacing(2), 
+    },
+    notifications: {
+        '& > * + *': {
+          marginTop: theme.spacing(2),
+        },
+        maxHeight: '500px',
+    },
+    title: {
+        flexGrow: 1,
+        paddingLeft: theme.spacing(1),
+    },
+    titleBox: {
+        display: 'flex',
+        alignItems: 'center',
+        paddingBottom: theme.spacing(1),
+    },
+}));
+
+export interface NotificationPopoverProps {
+    lastLoginTime: string;
+}
+
+export default function NotificationPopover({ lastLoginTime }: NotificationPopoverProps) {
+    const classes = useStyles();
+    const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+    const [analyses, setAnalyses] = useState<AnalysisRow[]>([] as AnalysisRow[]);
+    const popoverOpen = Boolean(anchorEl);
+
+    const handlePopoverOpen = (event: React.MouseEvent<HTMLButtonElement>) => {
+        setAnchorEl(event.currentTarget);
+    };
+    const handlePopoverClose = () => {
+        setAnchorEl(null);
+    };
+    useEffect(() => {
+        const lastLoginISO = new Date(lastLoginTime).toISOString().slice(0, -1);
+        fetch(`/api/analyses?since=${lastLoginISO}`).then(async response => {
+            if (response.ok) {
+                setAnalyses((await response.json()) as AnalysisRow[]);
```

Comment:
There's another step after fetching from https://github.com/ccmbioinfo/ST2020/blob/master/react/src/pages/analysis/Analysis.tsx#L177 to fit the `AnalysisRow` type.